e2e/pages/manageCases/caseProgression/reviewDocuments/fl401ReviewDocumentsSubmitPage.ts

```diff
@@ -0,0 +1,88 @@
+import { Page } from "@playwright/test";
+import { Selectors } from "../../../../common/selectors";
+import { Fl401ReviewDocumentsSubmitContent } from "../../../../fixtures/manageCases/caseProgression/reviewDocuments/fl401ReviewDocumentsSubmitContent";
+import { Helpers } from "../../../../common/helpers";
+import { CommonStaticText } from "../../../../common/commonStaticText";
+import AccessibilityTestHelper from "../../../../common/accessibilityTestHelper";
+import { yesNoDontKnow } from "../../../../common/types";
+
+interface FL401ReviewDocumentsSubmitPageOptions {
+  page: Page;
+  accessibilityTest: boolean;
+  yesNoNotSureReviewDocs: yesNoDontKnow;
+}
+
+export class FL401ReviewDocumentsSubmitPage {
+  public static async fl401ReviewDocumentsSubmitPage({
+    page,
+    accessibilityTest,
+    yesNoNotSureReviewDocs,
+  }: FL401ReviewDocumentsSubmitPageOptions): Promise<void> {
+    await this.checkPageLoads({
+      page,
+      accessibilityTest,
+      yesNoNotSureReviewDocs,
+    });
+    await this.fillInFields({ page });
+  }
+
+  private static async checkPageLoads({
+    page,
+    accessibilityTest,
+    yesNoNotSureReviewDocs,
+  }: Partial<FL401ReviewDocumentsSubmitPageOptions>) {
+    if (!page) {
+      throw new Error("No page found");
+    }
+    const optionsMap: Record<string, string> = {
+      yes: `${Selectors.GovukText16}:text-is("${CommonStaticText.yes}")`,
+      no: `${Selectors.GovukText16}:text-is("${CommonStaticText.no}")`,
+      dontKnow: `${Selectors.GovukText16}:text-is("${Fl401ReviewDocumentsSubmitContent.notSure}")`,
+    };
+
+    let yesNoNotSureSelector: string;
+    if (yesNoNotSureReviewDocs) {
+      yesNoNotSureSelector = optionsMap[yesNoNotSureReviewDocs];
+    } else {
+      throw new Error(
+        `Unknown value for yesNoNotSureReviewDocs: ${yesNoNotSureReviewDocs}`,
+      );
+    }
+    const testPdfSelector: string =
+      "body > exui-root > exui-case-home > div > exui-case-details-home > ccd-case-event-trigger > div > ccd-case-edit > ccd-case-edit-submit > div > form > div:nth-child(1) > table.form-table > tbody > tr:nth-child(1) > td.form-cell.case-field-content > ccd-field-read > div > ccd-field-read-label > div > ccd-read-dynamic-list-field > span";
```

Comment:
If you know what the PDF file name is going to be, can you just match on that instead? 

Something like:
```
page.locator('ccd-read-dynamic-list-field span', {hasText: 'testPdf.pdf'})
```